app/helpers/tutors_helper.rb

```diff
@@ -99,9 +100,15 @@ def affiliate
     ]
   end
 
+  def language_proficiencies
+    # Returns a hash like { "English => "None", "Spanish" => "None" }
+    Hash[(native_language.map { |e| e[0] }).collect { |v| [v, 'None'] }]
```

Comment:
Is the negative information (`"Portuguese" => "None"` for example) useful to keep since most tutors will have proficiency in only a couple of languages?